gpkit/nomials/nomial_core.py

```diff
@@ -61,7 +62,7 @@ def latex(self, excluded=None):
         if "units" in excluded:
             return " + ".join(sorted(mstrs))
 
-        units = unitstr(self.units, r"\mathrm{~\left[ %s \right]}", "L~")
+        units = self.unitstr(r"\mathrm{~\left[ %s \right]}", "L~")
         units_tf = units.replace("frac", "tfrac").replace(r"\cdot", r"\cdot ")
```

Comment:
replacement stuff...the latex formatting? Everywhere else calls upon Nomial to establish the proper formatting, so this is the only place it's specified